src/test/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/service/RoleServiceTest.kt

```diff
@@ -23,13 +24,14 @@ import uk.gov.justice.digital.hmpps.externalusersapi.config.AuthenticationFacade
 import uk.gov.justice.digital.hmpps.externalusersapi.jpa.repository.RoleRepository
 import uk.gov.justice.digital.hmpps.externalusersapi.model.AdminType
 import uk.gov.justice.digital.hmpps.externalusersapi.model.Authority
+import uk.gov.justice.digital.hmpps.externalusersapi.resource.CreateRole
 import uk.gov.justice.digital.hmpps.externalusersapi.resource.RoleAdminTypeAmendment
 import uk.gov.justice.digital.hmpps.externalusersapi.service.RoleService.RoleNotFoundException
 
 class RoleServiceTest {
   private val roleRepository: RoleRepository = mock()
   private val telemetryClient: TelemetryClient = mock()
-  private val authenticationFacade: AuthenticationFacade = AuthenticationFacade()
+  private val authenticationFacade: AuthenticationFacade = mock()
   private val roleService = RoleService(roleRepository, telemetryClient, authenticationFacade)
 
   init {
```

Comment:
This "init" section can probably be removed now - it was just being used for authenticationfacade which is now mocked